Essentials/src/com/earth2me/essentials/commands/Commandsethome.java

```diff
@@ -41,19 +42,41 @@ public void run(final Server server, final User user, final String commandLabel,
                 }
             }
         }
+
         if (checkHomeLimit(user, usersHome, name)) {
             name = "home";
         }
         if ("bed".equals(name) || NumberUtil.isInt(name)) {
             throw new NoSuchFieldException(tl("invalidHomeName"));
         }
+        if (ess.getSettings().isSetSameHomeByConfirm()) {
+            String uuid = user.getBase().getUniqueId().toString();
+            for (String h : usersHome.getHomes()) {
+                if (h.equalsIgnoreCase(name)) {
```

Comment:
Could potentially switch to TreeMap with CASE_INSENSITIVE_ORDER here

Reply:
I don't think it would make sense to create a separate TreeMap to use up the memory.

Reply:
Does this check happen other places? Are all home names case insensitive?